### DISCUSSION TITLE
Stepwise Planner in Copilot Chat App

### DISCUSSION THREAD

**Discussion by diptesh-bose** [microsoft/semantic-kernel#2395] [General]
Hi team, 

Is there a timeline for the release of stepwise planner in Copilot Chat App...

Thanks

**Comment by gitri-ms** [comment]
It's there now! You'll need to update the "Planner" config section in [appsettings.json](https://github.com/microsoft/chat-copilot/blob/main/webapi/appsettings.json) to specify "Stepwise" as your preferred planner.
